### ISSUE TITLE
Export types

### ISSUE THREAD

**Issue by Cyperghost** [WoltLab/editor#96]
The types should be automatically copied to the [WCF](https://github.com/WoltLab/WCF) repository in the `ts` directory. 
A similar workflow as for the [d.ts](https://github.com/WoltLab/d.ts) repository could be used for this.

```Bash
npx tsc --declaration --emitDeclarationOnly --outdir ./d.ts/

for folder in ./plugins/*/; do
  if [ -e "${folder}package.json" ]; then
    cp "${folder}package.json" "d.ts/${folder}"
  fi
done

mv ./d.ts/app.d.ts ./d.ts/ckeditor5-bundle.d.ts
```
The folders still need to be customized, this code is just an example.

This code creates the `d.ts` files once and copies the `package.json` so that the resolution of the modules works correctly.
Finally, the `app.d.ts` must be renamed, as it is available under `ckeditor5-bundle`.

**Comment by TimWolla** [comment]
This is resolved with WoltLab/WCF@13a0bc5b18999a1bc5de0575ece73d9d8bb6a5a4. The d.ts exporter of WoltLab/WCF will now automatically update the npm dependencies in the d.ts repository to match those in WoltLab/WCF. WoltLab/WCF itself relies on the original editor types since WoltLab/WCF@43fe9e527861822c94f956fd2280db130a02c39d.

TLDR: It should “just work” when updating to the latest version of the d.ts repository.
